dotcom-rendering/src/lib/getFrontsAdPositions.ts

```diff
@@ -181,26 +241,10 @@ const getFrontsBannerAdPositions = (
 		{ heightSinceAd: 0, adPositions: [] },
 	).adPositions;
 
-/**
- * Decides where ads should be inserted on tagged fronts.
- *
- * On tagged fronts, an ad in inserted above every third collection.
- *
- * Doesn't insert an ad above the final collection. We serve a merchandising slot below the
- * last collection and we don't want to sandwich the last collection between two full-width ads.
- */
-const getTaggedFrontsBannerAdPositions = (numCollections: number): number[] => {
-	if (numCollections <= 3) {
-		// There are no suitable positions to insert an ad.
-		return [];
-	}
-
-	const numAdsThatFit = Math.floor((numCollections - 1) / 3);
-
-	// Ensure we do not insert more than the maximum allowed number of ads.
-	const numAdsToInsert = Math.min(numAdsThatFit, MAX_FRONTS_BANNER_ADS);
-
-	return Array.from({ length: numAdsToInsert }, (_, i) => i * 3 + 2);
+export {
+	MAX_FRONTS_MOBILE_ADS,
+	MAX_FRONTS_BANNER_ADS,
```

Comment:
Do you want to re-export these constants?

Reply:
Nope! I missed this when creating the new constants file